diff --git a/sparse_dot_topn/__init__.py b/sparse_dot_topn/__init__.py
index cbaf32a..1ec9da0 100644
--- a/sparse_dot_topn/__init__.py
+++ b/sparse_dot_topn/__init__.py
@@ -1,7 +1,2 @@
 # flake8: noqa
-import sys
-
-if sys.version_info[0] >= 3:
-    from sparse_dot_topn.awesome_cossim_topn import awesome_cossim_topn
-else:
-    from awesome_cossim_topn import awesome_cossim_topn
\ No newline at end of file
+from sparse_dot_topn import awesome_cossim_topn
\ No newline at end of file
diff --git a/sparse_dot_topn/awesome_cossim_topn.py b/sparse_dot_topn/awesome_cossim_topn.py
index c4af03d..572150d 100644
--- a/sparse_dot_topn/awesome_cossim_topn.py
+++ b/sparse_dot_topn/awesome_cossim_topn.py
@@ -2,13 +2,8 @@ import sys
 import numpy as np
 from scipy.sparse import csr_matrix
 from scipy.sparse import isspmatrix_csr
-
-if sys.version_info[0] >= 3:
-    from sparse_dot_topn import sparse_dot_topn as ct
-    from sparse_dot_topn import sparse_dot_topn_threaded as ct_thread
-else:
-    import sparse_dot_topn as ct
-    import sparse_dot_topn_threaded as ct_thread
+import sparse_dot_topn as ct
+import sparse_dot_topn_threaded as ct_thread
 
 
 def awesome_cossim_topn(A, B, ntop, lower_bound=0, use_threads=False, n_jobs=1):
